doc/tasks/readme.md

```diff
@@ -0,0 +1,737 @@
+einer Aufgabe) und "Container" (um einen Questgegenstand anzunehmen). In solchen Fällen
+muss die Interaktion der einzelnen Komponenten durch die Taskbuilder-Methoden passend
+konfiguriert werden, um den Anforderungen durch den zu realisierenden Aufgabentyp gerecht
+zu werden.
+
+### Spielmechanik
+
+Eine Spielmechanik beschreibt die Interaktion von verschiedenen Entitätstypen im Spiel.
+Spielmechaniken beziehen sich ausschließlich auf die Interaktion von Spielelementen, die aus
+Game-Design Perspektive relevant sind (siehe [Steuermechanismen](#steuermechanismen) für
+alle anderen “Mechaniken”). Beispiel: ein Item, welches im Dungeon liegt, kann aufgehoben
+und in das Inventar des Spielcharakters transferiert werden.
+
+Aus einer Spielmechanik kann abgeleitet werden, welche Komponenten an der Mechanik beteiligt
+sind. Für obiges Beispiel sind folgende Komponenten erforderlich:
+
+- InventoryComponent im Spielcharakter (zur Realisierung des Spielerinventars)
+- CollisionComponent im Spielcharakter und im Item, um ein Event auszulösen, sobald der
+  Spielcharakter über das Item läuft
+- alternativ ein InteractionComponent im Item, falls das Item per dedizierter
+  Spielendenaktion (bspw. Tastendruck) aufgehoben werden soll
+
+### Spielszenario
+
+Ein Spielszenario beschreibt, welche Spielmechaniken, welche Entitätstypen und auf welche
+Art diese in einem konkreten Dungeonlevel in Verbindung gebracht werden, um eine abstrakte
+Aufgabe zu realisieren. Ein Spielszenario beschreibt nicht, welche visuellen
+
+Beispiel: In einem Raum befindet sich ein NPC und verschiedene Schalter. Der NPC stellt dem
+Spieler eine Frage und sagt ihm, welcher Hebel für welche Antwort steht.
+
+### Steuermechanismen
+
+Steuermechanismen sind alle Mechanismen des Systems, die den Ablauf der Umsetzung einer
+Aufgabe in ein Spielszenario beeinflussen.
+
+Steuermechanismen grenzen sich von Spielmechaniken ab, indem sie die übergeordneten
+Mechanismen realisieren, die sich nicht ausschließlich auf Game-Design Elemente beziehen.
+Hierzu zählen unter anderem das Logging von Antworten, die Organisation der Aufgabenfolge
+und Aufgabenverschachtelung.
+
+## Aufgabentypen
+
+### Single-Choice
+
+Die Aufgabenstellung besteht aus einem Fragetext und einer Menge an vorgegebenen
+Antwortmöglickeiten. Nur eine Antwort ist richtig. Die richtige Antwort muss ausgewählt
+werden.
+
+### Multiple-Choice
+
+Die Aufgabenstellung besteht aus einem Fragetext und einer Menge an vorgegebenen
+Antwortmöglickeiten. Mehrere Antworten sind richtig. Es müssen alle richtigen Antworten
+markiert werden. Es dürfen keine falschen Antworten markiert werden. Die Reihenfolge spielt
+keine Rolle.
+
+### Lückentext
+
+Die Aufgabenstellung besteht aus einem Fragetext. Die Antwort besteht aus einem oder
+mehreren Freitexten. Es muss jeweils die richtige Antwort in den Freitext-Stellen eingegeben
+werden. Die richtige Antwort kann ein konkreter Wert oder eine Eingabe passend zu einem
+regulären Ausdruck sein.
```

Comment:
```suggestion
regulären Ausdruck oder eine Auswahl aus einer vorgegebenen Menge an Antwortmöglichkeiten sein.
```

Reply:
@bcg7 @malt-r @AMatutat Ich finde die Beschränkung auf "Text" an der Stelle nicht so gut. Vom Konzept her kann ich in einem Lückentext auch andere Dinge einbauen, so lange ich die richtigen Bausteine an die richtige Stelle packe? Auch die Aufgabe muss nicht zwingend ein "Text" sein (auch wenn es "Lückentext" heisst), sondern könnte eine beliebige Struktur mit Lücken sein? Aber dann geht das irgendwann in "Zuordnen" über. Gedanken?

Reply:
Diese Diskussion haben wir schonmal so geführt und haben uns damals (genau aus dem Grund, dass die Abgrenzung Richtung "Zuordnen" schwierig wird) dafür entschieden, die Einschränkung beizubehalten.

Reply:
Stimmt. Da das immer wieder aufkommt, glaube ich, dass der Zuschnitt noch nicht richtig passend ist. Aber das ist ein Punkt für die allgemeine Diskussion und nicht für hier. Und vielleicht auch nicht mehr für dieses Projekt, wenn ich mir den Arbeitsstand so ansehe.

Reply:
@malt-r D.h. Lückentext ist wirklich nur ein freies Eingabefeld? Einen Lückentext mit vorgegebenen Antwortmöglichkeiten würdest Du als Zuordnung realisieren?

Reply:
Vielleicht mal aus GUI Sicht:
Wir hatten damals immer Freitext Aufgaben gesagt, aber Angela war damit irgendwie unglücklich und meinte das wären ja eher Lückentexte und irgendwie sind wir beim wording dabei geblieben aber lnhaltlich ist es ein Freitext. 
Nach allem was ich bisher über das GUI-LibGDX gedöns erfahren habe, scheint mir ein klassischer Lückentext auch sehr sehr schwierig/fummelig umsetzbar zu sein.

Reply:
🙈 Verstehe ich das richtig, dass wir hier was formulieren, was wir so gar nicht umsetzen (können)? 

Sollten wir dann entweder (a) die Beschreibung an die Wirklichkeit anpassen oder (b) das so lassen (und nicht umsetzen) und zusätzlich einen weiteren Aufgabentyp definieren, der auch umsetzbar ist?

Reply:
b)

Reply:
Wäre so ein Aufgabentyp dann nicht "Zuordnen"? Also ich ordne irgendwelche Elemente den Lücken in einer lückenhaften Struktur zu?

Reply:
> Wäre so ein Aufgabentyp dann nicht "Zuordnen"? Also ich ordne irgendwelche Elemente den Lücken in einer lückenhaften Struktur zu?

Ja. Aber hier gehts ja (wenn ich @AMatutat richtig verstehe) eigentlich um Freitext.

Reply:
Okay, dann könnte ich mir hier sowas wie "beschrifte Steintafeln und setz sie in die Lücken in der Wand ein" vorstellen. Also $n$ Freitext-Eingaben, die dann zusätzlich zu Lücken zugeordnet werden müssen. Der Typ könnte dann "Lücken füllen" heißen. Passt das für euch? Dann würde ich das mal hinzufügen.